pkg/uki/upgrade.go

```diff
@@ -64,6 +64,17 @@ func (i *UpgradeAction) Run() (err error) {
 		return err
 	}
 
+	// Check if the upgrade artifact contains the proper signature before copying
+	err = checkArtifactSignatureIsValid(i.cfg.Fs, filepath.Join(constants.UkiEfiDir, "EFI/Kairos", fmt.Sprintf("%s.efi", UnassignedArtifactRole)), i.cfg.Logger)
+	if err != nil {
+		// Remove efi file to not occupy space and leave stuff around
+		_ = removeArtifactSetWithRole(i.cfg.Fs, constants.UkiEfiDir, UnassignedArtifactRole)
```

Comment:
is there any value in loging the error instead of dismissing it?

Reply:
probably yes, will add